WooCommerce/Classes/Yosemite/AuthenticatedState.swift

```diff
@@ -90,6 +89,15 @@ class AuthenticatedState: StoresManagerState {
             JetpackConnectionStore(dispatcher: dispatcher)
         ]
 
+
+        if case let .wpcom(username: _, authToken: authToken, siteAddress: _) = credentials {
```

Comment:
Nit: this can be simplified:

```suggestion
        if case let .wpcom(_, authToken, _) = credentials {
```

Reply:
Done as part of b231bcef7029ee63fab0c1e723c9e645ed2e16e1